docs/docsite/rst/playbooks_filters.rst

```diff
@@ -318,6 +318,35 @@ address. For example, to get the IP address itself from a CIDR, you can use::
 More information about ``ipaddr`` filter and complete usage guide can be found
 in :doc:`playbooks_filters_ipaddr`.
 
+.. _network_filter:
+
+Network filters
+```````````````
+
+.. versionadded:: 2.4
+
+To generate a list of command diffs for a network configuration:
+
+  {{ myvar | diff_network_config(current_config, indent=1 }}
```

Comment:
need to close out the function: e.g. 
`{{ myvar | diff_network_config(current_config, indent=1 ) }}`

Also might be a good idea to show example using a file lookup.